tqdm/_tqdm.py

```diff
@@ -539,21 +539,20 @@ def inner(df, func, *args, **kwargs):
                     Data (may be grouped).
                 func  : function
                     To be applied on the (grouped) data.
-                *args, *kwargs  : optional
+                **kwargs  : optional
                     Transmitted to `df.apply()`.
                 """
+
                 # Precompute total iterations
                 total = getattr(df, 'ngroups', None)
```

Comment:
this `ngroups` is an inner attribute of pandas. It may be changed in future, but seems not happen in near future or never. So let's keep it so far. 

For perfection, I will suggest to use one of pandas public apis to get group size in future. So far, it is fine.